features/navbar.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Visitor Can Navigate via Navbar
+    As a visitor,
+    In order to navigate to different sections of the site,
+    I would like to be able to use a navbar
+
+  Scenario: Visitor Sees Navbar Links on Main Page
```

Comment:
This line should be indented the same as line 4
And then fix the indentation underneath when u have done that